### DISCUSSION TITLE
LDAP Auth "You are not authorized to login"

### DISCUSSION THREAD

**Discussion by mukhammart** [requarks/wiki#7363] [Help / Questions] [answered]
I've successfully connect LDAP to wikijs. Whenever I want to login via LDAP with my credentials of LDAP error occurs "You are not authorized to login". Seems that my login of LDAP is not attached to any existing group of wikijs.
The questions:
- How to get rid of such error?
- How to enable guest mode in wikijs, to check the problems with LDAP?

**Comment by NGPixel** [comment, marked as answer]
Have you enabled Self-registration in your LDAP config? Otherwise, users logging in for the first-time won't be able to have their wiki.js account created.

As for having guests, this is the default in new installs. Check your "Guests" group permissions and page rules.

**Reply by mukhammart** [reply]
Thanks, I've forgotten to close the discussion, since I could solve the issue by enabling Self-registration. I suppose term "Self-registration" should be changed to "Self-authorization" as it leads to misunderstanding. Because users already exist in LDAP server, therefore I could not catch it for the first time.

**Reply by NGPixel** [reply]
Self-authorization would be incorrect, as while they might exist in LDAP, they don't exist yet in Wiki.js. Self-registration allows a user that was authorized by the 3rd party authentication to have an account automatically created in Wiki.js if they don't already.

**Reply by mukhammart** [reply]
Thank you for the details. I did not know that a new account should be created in Wiki.js too.
